### DISCUSSION TITLE
Color saturation issues with Next image... no explanation

### DISCUSSION THREAD

**Discussion by danboyle8637** [vercel/next.js#24621] [Help]
Has anybody else noticed color saturation issues with Next Image?

What really makes this strange for me is it's only happening on occasions... on view of my app is desaturated and the next view is looking good... the next desaturated.

Each image component is set up the same so code wise I can't explain it.

I am hosting my images currently in Google storage... but I can't see how that matters since my public urls for the images look great.

Anybody find something similar going on?

**Comment by ugrupp** [comment]
I had a similar problem that would occur with next's default image optimization library in development: `squoosh`.

After installing `sharp`, the problem disappeared and the images were saturated, as expected.

https://nextjs.org/docs/basic-features/image-optimization#loader

**Comment by MatteoGauthier** [comment]
Got an issue with next 12.0.7
![image](https://user-images.githubusercontent.com/32040951/145689742-88161212-4f81-4bf6-be27-4e0ab3d21c29.png)

**Reply by alvara** [reply]
Hi @MatteoGauthier , im getting the same issue here on Next 12.0.9. 
I tried following this guide https://nextjs.org/docs/messages/sharp-missing-in-production but unfortunately still have the problem.
Were you ever able to figure out a solution?

**Reply by matteoveglia** [reply]
So I was having the exact same issue, I personally couldn't get sharp to work due to module.export errors (I'm new to NextJS) but I had a real facepalm moment when I realised it's due to an unrecognised colourspace used in the image file.

In my case my image, exported as a jpeg from Apple Photos used AdobeRGB (shot on my Canon 6D) but I simply re-exported from Photoshop with 'convert to sRGB' selected... et voila! no colour issues.

**Reply by beardsleym** [reply]
Thanks that was it! I had the same issue with `Colour profile: Adobe RGB (1998)` images.

**Comment by rdyar** [comment]
This seems to be an issue in development only - unless you are doing your own production builds and not using Sharp for image processing like the link above says.

It looks like the dev image processor is only expecting sRGB color space images so if you are using A98 or ProPhotoRGB or another color space your images will likely be out of whack in development.
 
If you deploy to vercel the rendered images there should look correct as it is using Sharp for processing the images.

**Comment by sjns19** [comment]
I've become the latest guy facing the same issue with Next 13. I have an image whose background color is also the background color for the web. I pick the color code off the image using the eye dropper tool in photoshop for the web's background and I can notice there is a significant difference between the image's and the web's background colors. This happens when I use Next's Image module. The problem goes away with html's img tag.

Tried the solution provided above converting the color profile to sRGB before exporting, still nothing. I decided to use the image as a background image in a CSS class instead of Next's Image for now.

This has to be sorted out, it's kind of annoying.

**Comment by g-a-v-i-n** [comment]
Same issue with P3 colorspace PNGs
